test/NumberToWords.spec.ts

```diff
@@ -12,6 +12,8 @@ it("numberToWords", () => {
 	expect(numberToWords(45, { ordinal: true })).toEqual("چهل و پنجم");
 	expect(numberToWords(0)).toEqual("صفر");
 	// @ts-ignore
-	expect(numberToWords()).toEqual("");
-	expect(numberToWords(502375902532527)).toEqual("");
+	expect(numberToWords()).toBeInstanceOf(TypeError);
+	expect(numberToWords(9006199254740992)).toEqual(
```

Comment:
Add more tests and also update the README to inform users that this function supports large numbers.